backend/internal/repository/get_image.go

```diff
@@ -0,0 +1,40 @@
+package repository
+
+import (
+	"fmt"
+	"log"
+
+	"math/rand"
+	"time"
+
+	"github.com/hoka-isdl/MERS/backend/internal/schema"
+)
+
+func GetImages() []schema.ListImagesInner {
+	var images schema.ListImagesInner
+	var imageslist []schema.ListImagesInner
+
+	rows_title, err := db.Query("SELECT id, google_drive_id FROM images")
+
+	if err != nil {
+		log.Fatal(err.Error())
+	}
+
+	fmt.Println(rows_title)
+	for rows_title.Next() {
+		rows_title.Scan(&images.ImageId, &images.GoogleDriveId)
```

Comment:
戻り値のerr忘れてる。
```suggestion
		err := rows_title.Scan(&images.ImageId, &images.GoogleDriveId)
```

FYI: 
Scanについて
https://pkg.go.dev/database/sql#Row.Scan